tests/generation/test_utils.py

```diff
@@ -680,6 +716,11 @@ def test_beam_sample_generate(self):
             else:
                 self.assertTrue(output_generate.shape[-1] == self.max_new_tokens + input_ids.shape[-1])
 
+            if any(name in model.__class__.__name__.lower() for name in ("blip", "llava", "paligemma")):
+                self.skipTest(
+                    "For VLMs inputs embeds won't match input ids unless images are encoded and merged with ids properly"
+                )
+
```

Comment:
This tags the whole test as skipped, which doesn't seem to be the goal (if the skip is reached, then the test above passes).

Let's simply run the rest of the test if a `if` condition is satisfied, and leave a TODO to work on these models

Reply:
oh no -- let's work it on a separate PR